api/organisations/views.py

```diff
@@ -260,9 +265,87 @@ def my_permissions(self, request, pk):
         return Response(serializer.data)
 
 
+def chargebee_webhook_payment_failed(request: Request) -> Response:
```

Comment:
I'd argue that this (and `chargebee_webhook_payment_success`) should be refactored into functions in e.g. the `chargebee` module that get called by the view here.

Reply:
Actually, I'd love to see us refactor all of this logic into some kind of `ChargebeeWebhookHandler` class  (with static methods maybe), or just a `chargebee_webhook_handler` module to really separate things out.

Reply:
Good idea, I've got something basic working here and I should have an updated PR by tomorrow.